src/scripts/upload.sh

```diff
@@ -11,14 +11,15 @@ if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
 if [ ! -r "$PATH_TO_LCOV" ]; then
```

Comment:
```
if [ ! -r "$PATH_TO_LCOV" ]; then
```
Hey @chapayevdauren I don't quite understand why this is determining whether we install via pip or npm. Testing the python image, for example, it tries to install coveralls via npm using this condition. Is `PATH_TO_LCOV` only available when coveralls is already installed via Python?

Reply:
PATH_TO_LCOV is just an environment variable pointing by default to this path `./coverage/lcov.info`. It is defined [here](https://github.com/ergeon/coveralls-orb/blob/develop/src/commands/upload.yml#L10C14-L10C34).
This file will exist when we collect coverage for node projects. For Python projects, it will not that is why I have added this condition. 
We could refactor and introduce another env variable which will be passed on CI config. Then, we can check in our script in which environment we are running it.
```shell
- coveralls/upload:
              is_node_env: true # by default false
```

Reply:
Hey @chapayevdauren, I tested the orb [here](https://github.com/ergeon/pipedrive-integration/pull/251/files) in pipedrive-integration, it seems to work fine. The issue in the context of the test is that we have to install coveralls first via pip for the `PATH_TO_LCOV` directory to exist. Every python test is defaulting to npm installation.

Reply:
we need to test it in python env without node mix.